react/features/toolbox/components/OpenUrlButton.js

```diff
@@ -0,0 +1,55 @@
+// @flow
+
+import { createToolbarEvent, sendAnalytics } from '../../analytics';
+import { translate } from '../../base/i18n';
+import { openURLInBrowser } from '../../base/util';
+import { AbstractButton, type AbstractButtonProps } from '../../base/toolbox';
+
+
+type Props = AbstractButtonProps & {
+
+    /**
+     * The URL to be opened.
+     */
+    _externalUrl: string,
+
+    /**
+     * A succinct description of what the button does.
+     */
+    accessibilityLabel: string;
+
+    /**
+     * The icon of this button.
+     */
+    icon: Object;
+
+    /**
+     * The text associated with this button. When `showLabel` is set to
+     * {@code true}, it will be displayed alongside the icon.
+     */
+    label: string;
+};
+
+/**
+ * Implements an {@link AbstractButton} to open a url in a new window.
+ */
+class OpenUrlButton extends AbstractButton<Props, *> {
+    /* eslint-disable no-invalid-this */
```

Comment:
If you define a constructor to assign those values I believe the eslint error will be fixed. Isn't it?